### DISCUSSION TITLE
Cloudflare captcha failed to bypass : inbetween

### DISCUSSION THREAD

**Discussion by slvamegestic** [seleniumbase/SeleniumBase#2517] [Q&A] [answered]
Hi All,
I also have the same issue like [#2414](https://github.com/seleniumbase/SeleniumBase/discussions/2414) while using ```reconnect_time=10``` it was working perfectly for sites while securing them at the time of the opening first page. But in some cases, like they have an captcha to verification in the middle of the processing our selenium base did not bypass that. Any ideas to bypass those cases?

**Comment by mdmintz** [comment, marked as answer]
Other than what's already mentioned in https://github.com/seleniumbase/SeleniumBase/discussions/2414#discussioncomment-8017964, there's no other advice I have for that.

If you need clicks to go undetected, try the UC Mode examples that click undetected:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

Those are currently working for me. If they aren't working for you, then your IP Address may have already been blacklisted.
